### PR TITLE
Update cyberint integration

### PR DESCRIPTION
**Pull request by TalGumi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by yucohen** [comment]
@TalGumi Thanks for your contribution! 
I'll review the PR in a couple of days. In the meantime, can you please fill out the contribution form?
Thanks again

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -105,4 +105,4 @@
 // 	"ghcr.io/devcontainers/features/common-utils:1"
 // ],
 "onCreateCommand": "dos2unix -n .devcontainer/createCommand.sh .devcontainer/createCommand_unix.sh && chmod +x .devcontainer/createCommand_unix.sh && bash .devcontainer/createCommand_unix.sh"
-}
\ No newline at end of file
+}
```
Comment (yucohen) [inline .devcontainer/devcontainer.json:108]: Please revert this file

**Review by yucohen — COMMENTED** [review]
@TalGumi Great work! 
@efelmandar Do you think we need a demo? (from the integration side I do not think it is needed)

**Comment by efelmandar** [comment]
Hi @TalGumi, thank you for your contribution! I have finished reviewing your PR and have noticed that most of the changes are to existing incident field types. Since this could potentially break BC for those using this content pack, creating new fields with the correct field type would be better, or using existing incident fields from the CommonTypes pack instead. Here are some incident fields from the CommonTypes pack that I think you can use: 

* Tags
* Created By
* Threat Actor
* Source Category
* Title
* Description
* Close Reason
* Ticket Number
* Category
* Alert ID
* External Confidence

Other than that, the PR looks good and we will be ready to merge once this has been addressed.
Please feel free to reach out here or in slack if you have any more questions or need any help :)

**Comment by yucohen** [comment]
Hey @TalGumi 
Seems like the only issue is with the validation that says: 

```
The 1_1_0.md release notes file contains incompatible Docker images:
- Cyberint: Release notes file has dockerimage: demisto/python3:3.10.10.48392 but the YML file has dockerimage: demisto/python3:3.10.10.49934
To fix this please run: 'demisto-sdk update-release-notes -i Packs/Cyberint'
```

Please let me know if you want me to fix this for you so we could merge :)

**Comment by yucohen** [comment]
@TalGumi I'll fix the validations in the internal PR as I don't have permissions to push on this branch

**Review by yucohen — APPROVED** [review]

**Pull request merged**